### ISSUE TITLE
UnamSanctam on Telegram?

### ISSUE THREAD

**Issue by alcinzal** [UnamSanctam/UnamWebPanel#398]
Hey Unam, so at the moment there are 68 members in the @UnamGroup on Telegram, and although the group isn't overflowing with activity 😅, we have still a nice community going. I, and many others, would be glad to see you in the group and on Telegram in general.

At the moment I am holding your username hostage as a channel, just to prevent someone from taking it and impersonating. If you ever decide you want to join Telegram, just let me know, so I can release the username.

If you do not wish to use your personal number, as it might be bad OPSEC practice, or you might already be using it for a personal Telegram account, I could help you out there. I bought 15 Telegram accounts about 6 months ago (I actually just needed 1, but there was a minimum amount), I have been using one of them personally with absolutely no problems, and I have checked the others and none have had any activity or been accessed.

If you wish to do this, I will send you an account, free of cost of course, together with a guide that will ensure that nobody can access the account, not me, not the seller, not even somebody with access to the number.

However if you don't wish to be on Telegram for other reasons, like that it'll open up another area where you need to answer questions and inquiries left and right, then I totally get that. But if you ever change your mind, I will release the username, and send you an account if required.

**Comment by amlkyc** [comment]
telegram is not that secure nowadays, I even got my bought account banned as many others, now rules are strict, especially about malware, I remember I could type "malware" in search and find different groups with knowladgable people, now there is nothing

**Comment by alcinzal** [comment]
> telegram is not that secure nowadays, I even got my bought account banned as many others, now rules are strict, especially about malware, I remember I could type "malware" in search and find different groups with knowladgable people, now there is nothing

Yeah I know, it really sucks. Telegram used to be the best place for communities like these. The accounts I have bought are still up though. They were registered with US numbers, I heard many accounts who had a chinese number got banned. Where was the number from that was used to register your account?
